setup.py

```diff
@@ -96,7 +95,10 @@ def find_files(pattern, path=None, root="rez"):
         "Programming Language :: Python",
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Rust",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
```

Comment:
I haven't reviewed this PR and I know it's still very WIP, but we don't really test nor advertise that it works with PyPy, so I'd better not add the PyPy classifier.